readme.md

```diff
@@ -118,7 +118,23 @@ Then, you can create additional user stories.
 
 > This strongly correlates to test-driven development (TDD), which we'll talk more about in future weeks.
 
-**You just created user stories for a duck.** That is: you defined what a user should experience in your creation in order for it to be considered a "complete" duck.
+**You just created some tentative user stories for a duck.** That is: you defined what a user should experience in your creation in order for it to be considered a "complete" duck.
+
+The convention that developers like to follow for user stories looks like this example of a facebook user story:
+
+As a user, I should be able to update my status, so that I can tell my friends and family
```

Comment:
The format is generally expressed as:

```
As a <role>, I want to <goal>, so that <reason>.
```

For this example, I'd update the goal and reason:

> As a user, I want to update my status, so that my friends and family can see what I'm thinking or doing.